src/igraph/sparse_matrix.py

```diff
@@ -4,7 +4,7 @@
 
 from __future__ import with_statement
 
-__all__ = ()
+__all__ = ['_graph_from_sparse_matrix']
```

Comment:
Let's not export `_graph_from_sparse_matrix`; you can still import it explicitly from other modules if you want to, but it does not become a part of the declared public API if you don't add it to `__all__`. (At least that's the general consensus, but nothing in Python enforces one to treat `__all__` as the declaration of the public API).

Reply:
Also, some linters complain if `__all__` is of a mutable type (like a list) because then you could in theory import `__all__` from elsewhere and append to it. So it's better to use a tuple to make the linters shut up.

Reply:
yep, agreed